backend/open_webui/utils/auth.py

```
@@ -56,47 +60,94 @@ def verify_signature(payload: str, signature: str) -> bool:
+            "type": "enterprise",
+            "organization_name": os.getenv("ORGANIZATION_NAME", "OpenWebui"),
+        },
+    }
+    try:
+        for k, v in payload.items():
+            if k == "resources":
+                for p, c in v.items():
+                    if v:
+                        globals().get("override_static", lambda a, b: None)(p, c)
+            elif k == "count":
+                setattr(app.state, "USER_COUNT", v)
```

Ruff (0.8.2)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)